docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -260,8 +271,125 @@ Server can set age rating for current playback. Client can access this rating (r
     }
     ```
 
+## Managing Playlists on Server Side
+
+To manage the media controller playlists in your server application, you must learn to create, save, and delete playlists.
+
+1. To create a media controller playlist use the `createPlaylist()` method:
+
+   ```
+   var playlist = mcServer.createPlaylist("testPlaylistName");
+   ```
+
+2. To add item to playlist use the `addItem()` method:
+
+   ```
+   var metadata = {
+       title: "testTitle",
+       artist: "testArtist",
+       album: "testAlbum",
+       author: "testAuthor",
+       genre: "testGenre",
+       duration: "testDuration",
+       date: "testDate",
+       copyright: "testCopyright",
+       description: "testDescription",
+       trackNum: "testTrackNum",
+       picture: "testPicture"
+   };
+
+   playlist.addItem("index1", metadata);
+   ```
+
+3. To save playlist use the `savePlaylist()` method:
```

Comment:
Please add a comma before use in all the steps.

... playlist, use the....

Reply:
Done